test/dshell/cpp_header_gen.d

```diff
@@ -4,22 +4,16 @@ import dshell;
 
 void main()
 {
-    if (OS != "linux")
-    {
-        writeln("CPP header generation test was skipped on non-linux platform.");
-        return;
-    }
-
-    // FIXME: Should be a default variable
-    Vars.set("CC", "g++");
-
     Vars.set("SOURCE_DIR",  "$EXTRA_FILES/cpp_header_gen");
-    Vars.set("LIB",         "$OUTPUT_BASE/lib$LIBEXT");
-    Vars.set("CPP_OBJ",     "$OUTPUT_BASE/cpp.o");
+    Vars.set("LIB",         "$OUTPUT_BASE/library$LIBEXT");
```

Comment:
This change isn't strictly necessary but seeing `lib.lib` could be pretty confusing for other readers.